datastore/unit_tests/test__gax.py

```diff
@@ -124,6 +124,69 @@ def _get_target_class():
     def _make_one(self, *args, **kwargs):
         return self._get_target_class()(*args, **kwargs)
 
+    def test_lookup(self):
+        from google.cloud.gapic.datastore.v1 import datastore_client
+
+        patch1 = mock.patch.object(
+            datastore_client.DatastoreClient, '__init__',
+            return_value=None)
+        patch2 = mock.patch.object(datastore_client.DatastoreClient, 'lookup')
+        patch3 = mock.patch(
+            'google.cloud.datastore._gax._grpc_catch_rendezvous')
+
+        with patch1 as mock_constructor:
+            ds_api = self._make_one()
+            mock_constructor.assert_called_once_with()
+            with patch2 as mock_lookup:
```

Comment:
@daspecster Using decorators won't work since I'm mock patching an object that gets imported at run time. `multiple` **kind of** works (maybe?) but that's really just for cases where you would use `patch(..., new='FOO')` and I don't have any `new` object to patch there (i.e. I just want a mock).